docs/DeveloperGuide.md

```diff
@@ -339,81 +348,426 @@ is better than these alternatives.<br>
 a. Include those libraries in the repo (this bloats the repo size)<br>
 b. Require developers to download those libraries manually (this creates extra work for developers)<br>
 
+
 ## Appendix A : User Stories
 
 Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (unlikely to have) - `*`
 
 
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
-`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the TaskBoss
+`* * *` | user | add tasks | record tasks that I need to get done |
+`* * *` | user | delete tasks | get rid of tasks that I no longer need
+`* * *` | user | edit tasks | update any outdated information
+`* * *` | user | see a list of all the tasks | view all my pending tasks in a single view
+`* * *` | user | mark my tasks as done | keep track of the task status
+`* * *` | user | set deadlines for tasks that have due dates | track the urgency of any given task
+`* * *` | user | sort the tasks based on their deadlines | know the most urgent tasks to address
+`* * *` | user | search for any task | find a task if I remember some keywords from the description
+`* *` | user | add a description to my tasks | keep note of the details related to any given task
+`* *` | user | undo a command | not worry about any accidental actions I do
+`* *` | user | export all the tasks into a specified folder to save | get access of my tasks from multiple computers
```

Comment:
would it be considered as an export operation or simply just setting the storage location?